pkg/apis/machineconfiguration/group.go

```diff
@@ -0,0 +1,6 @@
+// Package machineconfiguration contains machineconfiguration API versions.
+//
+// This file ensures Go source parsers acknowledge the machineconfiguration package
+// and any child packages. It can be removed if any other Go source files are
+// added to this package.
+package machineconfiguration
```

Comment:
OK, I'll read up on the doc

Reply:
I'm not sure if internal would work because then only the parents of the internal package could use it. At the moment, we need to use the machineConfig client only from the remediation controller, but I guess we'll also want to use the client to watch for pools from the suite controller. So I'm leaning towards the vendor directory, but there I need to check if the code generator would touch the vendor directory or if it has some special handling for vendor..

Reply:
Uhm... The vendor directory might be a problem for the rest of our dependencies. Since we use go-modules. Let's defer that to another iteration.